tests/test-definition.yaml

```diff
@@ -15,6 +15,9 @@ dimensions:
       - no-tls
       - insecure-tls
       - server-verification-tls
+  - name: opa
+    values:
+      - 0.66.0
```

Comment:
This is not a full review, just wanted to mention that the latest supported OPA version is now `1.0.1`

See https://github.com/stackabletech/docker-images/issues/998